src/server_manager/ui_components/outline-server-view.html

```diff
@@ -553,7 +553,7 @@ <h3>Create keys, share access</h3>
       getS3InviteUrl: function(accessUrl) {
         // If updating this URL, must also update it in outline-share-dialog.html.
         // TODO: deduplicate and move this to the TypeScript app code.
-        return 'https://s3.amazonaws.com/outline-vpn/index.html?admin_embed#/invite/' +
+        return 'https://s3.amazonaws.com/outline-vpn/invite.html?admin_embed#/invite/' +
```

Comment:
We could simplify the URL fragment format, but it would require changes to the HTML.  Currently, invite.html is just a copy of index.html, so it won't show an invite unless the URL fragment contains `/invite/`.

Reply:
That's definitely suboptimal. We should be able to keep things separate, but it seems that would require a big change. I'm a little unhappy with our website implementation, and would like to have that changed at some point.

It's ok to keep it as it is for now, but how hard will it be to change the url handling to invite.html without the hash in the future?

Reply:
I think it would be straightforward to make the code (which is identical in both copies) check its own URL and change behavior based on the filename.  That would allow us to construct URLs like 
```
https://s3.amazonaws.com/outline-vpn/invite.html/?admin_embed#/en/ss%3A%2F%2F...
```
Instead of the current
```
https://s3.amazonaws.com/outline-vpn/invite.html/?admin_embed#/en/invite/ss%3A%2F%2F...
```
If you think that's a worthwhile change, it's probably easy to implement.